torch-sys/libtch/torch_api.cpp

```diff
@@ -1280,6 +1281,17 @@ void ati_to_tensor_list(ivalue i,
   )
 }
 
+ivalue ati_object_method_(ivalue i, char *method_name, ivalue *ivalues, int nivalues) {
```

Comment:
Could we avoid using the same `i` name for the input ivalue and the for loop iteration below?

Reply:
I creatively renamed it to `j`. I agree on the change but I was mostly following existing patterns which used `i`.